packages/@aws-cdk/aws-lambda/test/event-source-mapping.test.ts

```diff
@@ -170,4 +170,46 @@ describe('event source mapping', () => {
       }],
     });
   });
+
+  test('throws if neither eventSourceArn nor kafkaBootstrapServers are set', () => {
```

Comment:
Besides these, add a test case that the new properties are rendered when pass in